honeybee_radiance/cli/schedule.py

```diff
@@ -0,0 +1,51 @@
+"""honeybee radiance schedule command."""
+import click
+import sys
+import logging
+
+from ladybug.epw import EPW
+
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group(help='Commands to create and modify schedules.')
+def schedule():
+    pass
+
+
+@schedule.command('epw-to-daylight-hours')
+@click.argument(
+    'epw', type=click.Path(exists=True, dir_okay=False, resolve_path=True))
+@click.option(
+    '--schedule-name', '-sn', help='Output file name for schedule.',
+    type=click.STRING, default='daylight_hours', show_default=True
+)
```

Comment:
If you are doing it like this, then you probably want to give the user bother the folder and the file name. We can also call it `name` instead of `schedule-name` since there are no other names. Here is an example:

https://github.com/ladybug-tools/honeybee-radiance/blob/f328bfcc2906afa8eb2aadffcbb8f1f4476b060f/honeybee_radiance/cli/sky.py#L156-L157